Frontend/src/Components/Home/Home.jsx

```diff
@@ -44,9 +44,9 @@ const Home = () => {
         <div className="styleSection">
           <h3>Shop by Style</h3>
           <div className="styleItems">
-            <div className="styleItem"></div>
-            <div className="styleItem"></div>
-            <div className="styleItem"></div>
+            <div className="styleItem">Dresses</div>
```

Comment:
nit: for commits just for the back-end, make sure not to commit any front-end code! commit these front-end changes separately